### ISSUE TITLE
Ptt-CW delay with straight keyer

### ISSUE THREAD

**Issue by hb9bun** [ExpertSDR3/ExpertSDR3-BUG-TRACKER#609]
SunSDR2 DX, EESDR 1.0.4 beta, Build 24042023, Win 11:  
I expected the Ptt-CW delay with a straight keyer under Break.In to introduce a delay between PTT and the CW signal of up to 50ms. 
Working with an external Winkeyer keying the SunSDR2 DX as a straight keyer, the effect with Ptt-CW delay is only that the each dot and dash is being cut, e.g. as heard in the monitor, but PTT to CW signal is not delayed. It seems like that delay for Straight keyers does not work yet. 
The straight keyer is connected with ring open.
Just nice to have: A delay up to 100ms (not only 50) would be nice. 
73, Peter - HB9BUN

**Comment by alexloza1965** [comment]
I agree with you.  Looks like Ptt-CW Delay is not working as it should. And yes, 100 mc would be OK. But it should be between PTT and signal out.
